### DISCUSSION TITLE
Welcome to rikaichamp discussions!

### DISCUSSION THREAD

**Discussion by birtles** [birchill/10ten-ja-reader#465] [General]
## 👋 ようこそ～
Hi fellow Japanese reader!

If you have:

* A question about rikaichamp ❓
* An idea you want to discuss 💡
* Anything else that doesn't make sense to file as an issue 🤔

Please post away here! ✍️

The people using Rikaichamp have always been incredibly positive and supportive. Let's keep it that way. よろしくお願いします🙇‍♂️

**Comment by QuickOats** [comment]
What database does Rikaichamp use for definitions?

**Reply by birtles** [reply]
Hi! Rikaichamp is largely based on JMdict/DICT, JMNedict/ENAMDICT, KANJIDIC but it also draws in some other sources such as pitch accent info and some custom kanji component information and tweaks. You can see details in the Rikaichamp options page along with links to the different resources.

**Comment by hikarutanjou** [comment]
Hello Brian!

Just want to give a word of thanks to you for creating this amazing tool for FireFox. I used to use Rikaikun on Chrome, but I didn't really want to continue using Chrome for a variety of reasons. Lots of really neat features with Rikaichamp, and it has helped me immensely as a tool while translating. Instead of using various dictionaries on my second monitor when I come across some difficult phrases or really technical kanji, I just hover over them, and viola! The blue square helps me speed up my translation process so much. Thank you again! If you had a developer tip jar or something, I would love to send at least a coffee your way.

Thanks!

Joshua Fletcher

**Reply by birtles** [reply]
Hi Joshua!

Thank you so much! That's incredibly encouraging feedback!

I don't have a developer tip jar at the moment but either a follow on twitter ([@rikaichamp](https://twitter.com/rikaichamp)) or a review on the Chrome Web Store / Firefox Add-ons site etc. is very much appreciated!

Thank you once again!

Brian Birtles
